pkg/controller/v1alpha2/experiment/experiment_controller.go

```diff
@@ -233,7 +232,6 @@ func (r *ReconcileExperiment) Reconcile(request reconcile.Request) (reconcile.Re
 			logger.Error(err, "Create experiment in DB error")
 			return reconcile.Result{}, err
 		}
-		requeue = true
```

Comment:
why is this removed?

Reply:
in this loop, instance status changed or fails(line 234). for status changed case, there is no need to requeue since changes of obj will make it into work queue again; for db write fail case, the obj key will not be pop out of the queue, similar as `Requeue: true`

Reply:
Can you check if reconcile function will be called immediately or wait for the sync period?

Reply:
if `Requeue` is true even update done, it will be called immediately. Then we calls Get operation, the return nearly always is obsolete data (That's why I find I need do this change).